test/unit-tests/azure_gpu_ecc_sdbe.sh

```diff
@@ -40,12 +40,18 @@ ppending_blacklist_sections=($(awk '/Pending Page Blacklist / {print $NF}' $test
     set +e
```

Comment:
Is this entire file needed since we moved the unit tests into the other file?

Reply:
I guess not, I can remove it.